engine/table/src/main/java/io/deephaven/engine/tablelogger/impl/memory/MemoryTableLogger.java

```diff
@@ -15,7 +15,11 @@
 
 /**
  * Base class for memory table loggers that create and initialize a {@link DynamicTableWriter}.
+ *
+ * <p>
+ * Deprecated: prefer constructions using blink tables, see {@link io.deephaven.stream.StreamToBlinkTableAdapter}.
  */
+@Deprecated(since = "0.26.0", forRemoval = true)
```

Comment:
What still uses this?

Reply:
Nothing in DHC, but checking w/ DHE. Technically, users could too.